src/test/java/com/gdsc/studyjamapi/service/UserServiceTests.java

```diff
@@ -160,4 +163,87 @@ void getCurrentUserResponse_returnOk() {
     assertEquals(expecteUser.getId(), actualResponse.getId());
     assertEquals(expecteUser.getEmail(), actualResponse.getEmail());
   }
+
+  @Test
+  void CanCreateAndEditAccount() {
+    CreateUserRequest request = CreateUserRequest.builder()
+            .fullName("Full Name")
+            .email("test@gmail.com")
+            .password("password")
+            .role("SUPER_ADMIN")
+            .build();
+
+    when(userRepository.findUserByEmail("test@gmail.com")).thenReturn(Optional.empty());
+
```

Comment:
Could you please tell me why you created a mock to return an empty Optional here but also another to return an Optional with a mock user below? If you want to test the exception, you have to create another unit test.